js/lesson-24/part10.js

```diff
@@ -0,0 +1,18 @@
+`use strict`;
+
+let startOfRange = 0;
+let engOfRange = 100;
+let win = false;
+
+while (!win) {
```

Comment:
Algorithm is not fully working,  it stucks in loop and number can't be guessed correctly

Reply:
Fixed